### ISSUE TITLE
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process..

### ISSUE THREAD

**Issue by naqoyqatsi83** [SublimeLinter/SublimeLinter#1829]
Hello,

I am trying to use salt-lint in combination with SublimeLintner in Sublime Text.. but I am running to this issue:
```
SublimeLinter: log_handler.py:74      Logging installed; log level INFO
SublimeLinter: sublime_linter.py:74   debug mode: on
SublimeLinter: sublime_linter.py:75   version: 4.17.0
SublimeLinter: sublime_linter.py:345  Delay linting 'init.sls' for 0.0s
SublimeLinter: linter.py:1127         salt-lint: Checking lint mode 'manual' vs lint reason 'on_load'.  Skip.
Package Control: Skipping automatic upgrade, last run at 2021-10-27 08:20:34, next run at 2021-10-27 09:20:34 or after
SublimeLinter: linter.py:1127         salt-lint: Checking lint mode 'manual' vs lint reason 'on_user_request'.  Ok.
SublimeLinter: sublime_linter.py:345  Delay linting 'init.sls' for 0.0s
SublimeLinter: linter.py:1127         salt-lint: Checking lint mode 'manual' vs lint reason 'on_user_request'.  Ok.
SublimeLinter: #1 linter.py:1144      salt-lint: linting 'init.sls'
SublimeLinter: #1 util.py:161         PATH:
    C:\Users\pg\AppData\Roaming\Python\Python37\Scripts
    C:\Windows\system32
    C:\Windows
    C:\Windows\System32\Wbem
    C:\Windows\System32\WindowsPowerShell\v1.0\
    C:\Windows\CCM
    C:\Windows\CCM
    C:\Users\pg\AppData\Local\Microsoft\WindowsApps
    C:\Users\pg\AppData\Local\Programs\Git\cmd
SublimeLinter: #1 linter.py:1699      Running ...

  C:\Users\pg\tc  (working dir)
  > type saltmaster.git\srv\salt\common\states\oracle\init.sls | C:\Users\pg\AppData\Roaming\Python\Python37\Scripts\salt-lint.EXE --nocolour --severity

SublimeLinter: #1 linter.py:808       WARNING: salt-lint output:
Traceback (most recent call last):
  File "c:\program files\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\program files\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\pg\AppData\Roaming\Python\Python37\Scripts\salt-lint.EXE\__main__.py", line 9, in <module>
  File "C:\Users\pg\AppData\Roaming\Python\Python37\site-packages\saltlint\cli.py", line 72, in run
    os.unlink(stdin_file.name)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\pg\\AppData\\Local\\Temp\\tmp_9l1dl48.sls'

SublimeLinter: #1 linter.py:810       Note: above warning will become an error in the future. Implement `on_stderr` if you think this is wrong.
SublimeLinter: #1 linter.py:1220      salt-lint: no output
SublimeLinter: sublime_linter.py:593  Linting 'init.sls' took 1.38s
```

here are my settings:

```
{
    "debug": true,
    "lint_mode": "manual",
    "paths": {
        "linux": [],
        "osx": [],
        "windows": ["C:\\Users\\pg\\AppData\\Roaming\\Python\\Python37\\Scripts"]
    }
}
```

I've checked permissions on directory.. I don't see any issue.. even those temporary files are readable by my user.. process is running under my user.. So from this point of view I don't see any reason for such  error.. 

.. any ideas what might be wrong?

**Comment by roaldnefs** [comment]
This issue can be closed as the related issue warpnet/salt-lint#262 has been fixed in warpnet/salt-lint#263 which was included in salt-lint version 0.7.0 (_released 6 days ago_).

**Comment by kaste** [comment]
Thanks for letting us know.
